### PR TITLE
Namespace scope ui

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/actions/blackducks.go [hunk 1]
```diff
@@ -231,25 +225,20 @@ func (v BlackducksResource) Create(c buffalo.Context) error {
...
-	}
-
-	ns, err := util.CreateNamespace(v.kubeClient, blackduck.Spec.Namespace)
 	if err != nil {
-		return v.redirect(c, blackduck, err)
+		err = util.DeployCRDNamespace(v.kubeConfig, v.kubeClient, util.BlackDuckName, blackduck.Spec.Namespace, blackduck.Name, blackduck.Spec.Version)
+		if err != nil {
+			return v.redirect(c, blackduck, err)
+		}
+		log.Infof("created %s Black Duck instance in %s namespace", blackduck.Name, blackduck.Spec.Namespace)
```
Comment (jdartigalongue) [inline cmd/operator-ui/actions/blackducks.go]: Should we slightly change the log message? Only the namespace has been created at this stage so "created namespace %s for Black Duck instance %s" would be more appropriate here.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#pullrequestreview-250441443" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#pullrequestreview-250441443" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/460#pullrequestreview-250497426" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/actions/blackducks.go (hunk 1 above)
Comment (msenmurugan) [reply cmd/operator-ui/actions/blackducks.go]: Sorry didn't see your comment. Will fix it in next PR

**Pull request merged**